sql/data_model.sql

```diff
@@ -0,0 +1,64 @@
+BEGIN;
+
+-- add pgcrypto extension so that we can use the gen_random_uuid function.
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
+
+DROP TABLE IF EXISTS formdata CASCADE;
+CREATE TABLE formdata(
+  id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+  domain text NOT NULL,
+  received_on timestamp without time zone,
+  time_start timestamp without time zone,
+  time_end timestamp without time zone,
+  duration bigint,
```

Comment:
Generally agree with this but bear in mind that every additional check
/constraint uses db resources.